net-mgmt/lldpd/src/opnsense/service/templates/OPNsense/Lldpd/lldpd

```diff
@@ -0,0 +1,8 @@
+{% if helpers.exists('OPNsense.lldpd.general.enabled') and OPNsense.lldpd.general.enabled == '1' %}
+lldpd_opnsense_bootup_run="/usr/local/opnsense/scripts/OPNsense/Lldpd/setup.sh"
```

Comment:
this one can be omitted, was renamed to _var_script :)

Reply:
Can you give me an example please ...

Reply:
just delete the line, everything else is fine

Reply:
OK, done